### PR TITLE
Implemented dark mode

### PR DESCRIPTION
**Pull request by Palakkgoyal**
This PR closes issue no. #541 
Screenshot 2023-06-30 133147
Screenshot 2023-06-30 133228
Screenshot 2023-06-30 133303
Screenshot 2023-06-30 133322

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -54,8 +54,8 @@
   <body>
     <div id="root">
       <script type="module" src="/src/main.jsx"></script>
-      <script src="https://cdn.jsdelivr.net/npm/darkmode-js@1.5.7/lib/darkmode-js.min.js"></script>
       <script src="/src/script.js" type="module"></script>
+      <script src="/src/js/darkMode.js" type="module"></script>
```
Comment (narayan954) [inline index.html:58]: this file doesn't exists

**Review by narayan954 — CHANGES_REQUESTED** [review]
You did really good, but some of the styles are missing such as the shadow effects on logout modal
image
Signup and signin pages have also lost their cover
image

Also, I've applied background color from github dark mode
image
Is that okay??
Please fix those lost styles and I can update the difficulty level of this issue :)

**Comment by Palakkgoyal** [comment]
Yes, sure. I will do the required changes.

**Comment by Palakkgoyal** [comment]
Did all the changes. Please review the
Screenshot (565)
Screenshot (566)
Screenshot (567)
m.

**Review by narayan954 — APPROVED** [review]
looks good to me!

**Comment by narayan954** [comment]
> 
> @narayan954 Dont give level3 tags to such basic work. it is atmost level2

it's level 2 only, might've assigned level2 by mistake

**Pull request merged**